Core/src/org/sleuthkit/autopsy/casemodule/services/FileManager.java

```diff
@@ -510,26 +511,26 @@ private static synchronized String generateFilesDataSourceName(SleuthkitCase cas
      * @throws TskCoreException If there is a problem completing a database
      *                          operation.
      */
-    private AbstractFile addLocalFile(CaseDbTransaction trans, VirtualDirectory parentDirectory, java.io.File localFile,
+    private AbstractFile addLocalFile(CaseDbTransaction trans, AbstractFile parentDirectory, java.io.File localFile,
```

Comment:
Perhaps use SpecialDirectory instead of AbstractFile as teh type for the parentDirectory parameter?